x/twap/types/utils.go

```diff
@@ -6,30 +6,26 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
-	"github.com/osmosis-labs/osmosis/v12/osmoutils"
 )
 
 var MaxSpotPrice = sdk.NewDec(2).Power(128).Sub(sdk.OneDec())
 
-// GetAllUniqueDenomPairs returns all unique pairs of denoms, where for every pair
```

Comment:
Part of the godoc was removed. Please add it back. Logic LGTM

Reply:
I brought them back